cpp/purify/yaml-parser.h

```diff
@@ -85,50 +84,52 @@ class YamlParser
   private: \
   TYPE NAME##_ =  VALUE; \
   public: \
-  TYPE NAME(){ return NAME##_;};
-
-  YAML_MACRO(std::string, filepath, "");
-  YAML_MACRO(std::string, timestamp, "");
-  YAML_MACRO(std::string, logging, "");
-  YAML_MACRO(std::string, algorithm, "");
-  YAML_MACRO(std::string, mpiAlgorithm, "");
-  YAML_MACRO(std::string, param1, "");
-  YAML_MACRO(std::string, param2, "");
-  YAML_MACRO(bool, realValueConstraint, true);
-  YAML_MACRO(bool, positiveValueConstraint, true);
-  YAML_MACRO(int, iterations, 0);
-  YAML_MACRO(int, powMethod_iter, 0);
-  YAML_MACRO(int, x, 0);
-  YAML_MACRO(int, y, 0);
-  YAML_MACRO(int, wavelet_levels, 0);
-  YAML_MACRO(int, epsilonConvergenceScaling, 0);
-  YAML_MACRO(unsigned int, Jx, 0);
-  YAML_MACRO(unsigned int, Jy, 0);
-  YAML_MACRO(float, oversampling, 0);
-  YAML_MACRO(float, powMethod_tolerance, 0);
-  YAML_MACRO(float, chirp_fraction, 0);
-  YAML_MACRO(float, kernel_fraction, 0);
-  YAML_MACRO(double, epsilonScaling, 0);
-  YAML_MACRO(double, Dx, 0);
-  YAML_MACRO(double, Dy, 0);
-  YAML_MACRO(double, relVarianceConvergence, 0);
-  YAML_MACRO(std::vector<std::string>, wavelet_basis, {});  
-  YAML_MACRO(std::string, gamma, "");
-  YAML_MACRO(std::string, output_path, "");
-
-  YAML_MACRO(purify::utilities::vis_source, source, purify::utilities::vis_source::measurements);
-  YAML_MACRO(std::vector<std::string>, measurements_files, {});
-  YAML_MACRO(std::string, measurements_polarization, "");
-  YAML_MACRO(purify::utilities::vis_units, measurements_units, purify::utilities::vis_units::radians);
-  YAML_MACRO(float, measurements_sigma, 1);
-  YAML_MACRO(std::string, skymodel, "");
-  YAML_MACRO(float, signal_to_noise, 30);
-  
-  YAML_MACRO(std::string, Jweights, "");
-  YAML_MACRO(bool, wProjection, true);
+  const TYPE NAME(){ return NAME##_;};
+
+  YAML_MACRO(std::string, filepath, "")
+  YAML_MACRO(std::string, timestamp, "")
+  YAML_MACRO(std::string, logging, "")
+  YAML_MACRO(std::string, algorithm, "")
+  YAML_MACRO(factory::algo_distribution, mpiAlgorithm,
+      factory::algo_distribution::serial)
+  YAML_MACRO(purify::utilities::vis_source, source, purify::utilities::vis_source::measurements)
+  YAML_MACRO(std::string, param1, "")
+  YAML_MACRO(std::string, param2, "")
+  YAML_MACRO(bool, realValueConstraint, true)
+  YAML_MACRO(bool, positiveValueConstraint, true)
+  YAML_MACRO(int, iterations, 0)
+  YAML_MACRO(int, powMethod_iter, 0)
+  YAML_MACRO(int, x, 0)
+  YAML_MACRO(int, y, 0)
+  YAML_MACRO(int, wavelet_levels, 0)
+  YAML_MACRO(int, epsilonConvergenceScaling, 0)
+  YAML_MACRO(unsigned int, Jx, 0)
+  YAML_MACRO(unsigned int, Jy, 0)
+  YAML_MACRO(float, oversampling, 0)
+  YAML_MACRO(float, powMethod_tolerance, 0)
+  YAML_MACRO(float, chirp_fraction, 0)
```

Comment:
Remove wProjection, chirp_fraction, kernel_fraction, since they were removed from the cc.

Reply:
I agree

Reply:
done